### PR TITLE
Dbz 666- Support Ordered snapshotting of data.

### PR DESCRIPTION
**Pull request by vamossagar12**

### CODE REVIEW

**Comment by jpechane** [comment]
@vamossagar12 Hi, could you please add a test for the feature? Also the message for the commit should be probably more descriptive for the change. Thanks!

File: debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java [hunk 1]
```diff
@@ -647,6 +646,14 @@ public static EventProcessingFailureHandlingMode parse(String value) {
                                                       .withValidation(MySqlConnectorConfig::validateColumnBlacklist)
                                                       .withDescription("");
 
+    public static final Field TABLE_SNAPSHOT_ORDER_SPECIFIER = Field.create("table.snapshot.order.specifier")
+                                                                .withDisplayName("Snapshot table data in the specified order")
```
Comment (rhauch) [inline debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java]: The display name should be something like "Explicit Snapshot Table Order".

File: debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java [hunk 2]
```diff
@@ -647,6 +646,14 @@ public static EventProcessingFailureHandlingMode parse(String value) {
...
+                                                                .withType(Type.STRING)
+                                                                .withWidth(Width.LONG)
+                                                                .withImportance(Importance.LOW)
+                                                                .withValidation(MySqlConnectorConfig::validateTableSnapshotOrderSpecifier)
+                                                                .withDescription("");
```
Comment (rhauch) [inline debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java]: The description should be something you might find in a tool tip or in generated documentation: what does this configuration property do?

**Comment by gunnarmorling** [comment]
As discussed in the chat and the mailing list a while ago, we could simply use the existing table whitelist option to derive an order (if an explicit order for all tables is needed, N non-wildcard patterns for N tables would have to be given. I.e. within the group of tables matching a wildcard pattern, no order will be defined).

**Comment by gunnarmorling** [comment]
Hey @vamossagar12, how is it going with this one? Are you planning to follow up on the last comments? Thanks!

**Comment by vamossagar12** [comment]
hey @jpechane as of now I didn't get a chance to implement the discussed changes. I plan to start soon though. Would that be ok?

**Comment by vamossagar12** [comment]
hey @gunnarmorling I am closing this PR in support of this one : https://github.com/debezium/debezium/pull/507

**Pull request closed without merging**